### ISSUE TITLE
Get auto claims data from Charles

### ISSUE THREAD

**Issue by sjwoodard** [DuffAndPhelps/demo-auto-claims#1]
Need data set from claims team for the problem

**Comment by sjwoodard** [comment]
sample comment

**Comment by vijay-agrawal** [comment]
need more data

**Comment by joeldlawler** [comment]
i have commented.

![image](https://user-images.githubusercontent.com/7420032/70540094-a217b980-1b32-11ea-80c0-593d9f6b29f6.png)

**Comment by suvek49** [comment]
training dataset is needed

**Comment by sjwoodard** [comment]
@charleswm can you help us get this data to me ASAP?

**Comment by vijay-agrawal** [comment]
i need documentation

**Comment by chandrayan0011** [comment]
@sjwoodard Please help with Training data :P

**Comment by charleswm** [comment]
> @charleswm can you help us get this data to me ASAP?

Yeah, let's upload it as a dataset to azureml. It's a pretty small file, so we don't need to build a full data pipeline to get started.
